cmd/util/ledger/migrations/account_migration.go

```diff
@@ -0,0 +1,94 @@
+package migrations
+
+import (
+	"fmt"
+
+	"github.com/onflow/flow-go/fvm/environment"
+	"github.com/onflow/flow-go/fvm/meter"
+	"github.com/onflow/flow-go/fvm/state"
+	"github.com/onflow/flow-go/ledger"
+)
+
+func MigrateAccountUsage(payloads []ledger.Payload, pathFinderVersion uint8) ([]ledger.Payload, []ledger.Path, error) {
+	return MigrateByAccount(AccountUsageMigrator, payloads, pathFinderVersion)
+}
+
+func KeyToStorageInteractionKey(key ledger.Key) (meter.StorageInteractionKey, error) {
+	id, err := KeyToRegisterID(key)
+	if err != nil {
+		return meter.StorageInteractionKey{}, err
+	}
+	return meter.StorageInteractionKey{
+		Owner: id.Owner,
+		Key:   id.Key,
+	}, nil
+}
+
+func payloadSize(key ledger.Key, payload ledger.Payload) (uint64, error) {
+	ik, err := KeyToStorageInteractionKey(key)
```

Comment:
we can't directly get the cached storage size of a given payload. Instead, the method to get storage size of a payload requires a storage interaction key, that's why we need to first convert the key into a storage interaction key here.